src/index.js

```diff
@@ -82,7 +83,14 @@ function Index() {
     <Provider store={store}>
       <MuiThemeProvider theme={theme}>
         <Router>
-          <App />
+          <SnackbarProvider
```

Comment:
Can we put the router inside all the providers? Just for organization's sake.

Reply:
Sorry, I already merged before I saw this. I can do it in my next ticket, which is also snackbar